hcn/hcnpolicy.go

```diff
@@ -143,6 +167,13 @@ type L4ProxyPolicySetting struct {
 	ExceptionList []string `json:",omitempty"`
 	Destination   string   `json:","`
 	OutboundNat   bool     `json:",omitempty"`
+
+	// For the WFP proxy
+	FilterTuple          FiveTuple `json:",omitempty"`
+	ProxyType            ProxyType `json:",omitempty"`
+	ID                   guid.GUID `json:",omitempty"`
```

Comment:
do we use guid.GUID elsewhere?

Reply:
It's used a few times across the repo [[1]](https://github.com/microsoft/hcsshim/blob/e5b7236/hcn/hcnloadbalancer.go#L56) [[2]](https://github.com/microsoft/hcsshim/blob/e5b7236/hcn/hcnendpoint.go#L65) [[3]](https://github.com/microsoft/hcsshim/blob/e5b7236/cmd/runhcs/container.go#L55), thoughts?

Reply:
Hm, those are mainly internal functions. I think I'd prefer Id as a string as that follows the convention of most of our structs, but do whats easiest/makes sense to you.

Reply:
Hey, good news. You can go ahead and remove the ID completely. It's not needed.